qiskit/synthesis/one_qubit/__init__.py

```diff
@@ -0,0 +1,15 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017 - 2023.
```

Comment:
I'm always a bit confused by the copyright. We mostly have `2017, <year of file creation>` separated by the comma. The dash seems to indicate that the copyright would expire in 2023, which I guess we don't want (but again I'm no expert 🙂)